backend/bittan/bittan/views/views.py

```diff
@@ -23,3 +23,38 @@ def get_chapterevents(request: Request) -> Response:
     chapterevents = ChapterEvent.objects.filter(sales_stop_at__gt=now).order_by("event_at")
     s = ChapterEventSerializer(chapterevents, many=True)
     return Response(s.data)
+
+class ValidateValidateTicketSerializer(serializers.Serializer):

```

Comment:
Change name to ValidateTicketRequestSerializer

Reply:
a3c75a6198aea9f6a5ea79000b28ff6c652d40b4